.github/gh_runner_dependencies.sh

```diff
@@ -35,6 +35,7 @@ tfenv install 0.14.9
 tfenv install 0.15.5
 tfenv install 1.0.0
 tfenv install 1.0.1
+tfenv install 1.0.6
```

Comment:
@Shabirmean you may have to rerun this on the self hosted runners

Reply:
This has been updated